syntax/list.php

```diff
@@ -53,9 +57,34 @@ public function connectTo($mode) {
      * @return array Data for the renderer
      */
     public function handle($match, $state, $pos, Doku_Handler $handler){
-        $data = array();
+        global $conf;
+
+        $lines = explode("\n", $match);
+        array_shift($lines);
+        array_pop($lines);
+
+        try {
+            $parser = new ConfigParser($lines);
+            $config = $parser->getConfig();
+            return $config;
+        } catch(StructException $e) {
+            msg($e->getMessage(), -1, $e->getLine(), $e->getFile());
+            if($conf['allowdebug']) msg('<pre>' . hsc($e->getTraceAsString()) . '</pre>', -1);
+            return null;
+        }
+    }
 
-        return $data;
+    /**
+     * Checks for options that do not work in a list aggregation
+     *
+     * @param array $config
+     */
+    protected function checkForInvalidOptions($config) {
+        $illegal = ['dynfilters', 'summarize', 'rownumbers', 'widths', 'summary'];
+        foreach ($illegal as $illegalOption)
+        if (!empty($config[$illegalOption])) {
+            throw new StructException('illegal option', $illegalOption);
+        }
```

Comment:
I guess we need this for the other syntax modes. Eg. sepbyheaders is not available for table or lookup.